aptos-move/aptos-transaction-benchmarks/src/transaction_bench_state.rs

```diff
@@ -0,0 +1,326 @@
+// Copyright © Aptos Foundation
+
+use crate::{transactions, transactions::RAYON_EXEC_POOL};
```

Comment:
This is just moving the `TransactionBenchState` to a new module, no logic change.